pkg/kubelet/dockertools/exec.go

```diff
@@ -167,3 +178,37 @@ func (*NativeExecHandler) ExecInContainer(client DockerInterface, container *doc
 
 	return err
 }
+
+func cgexecArgs(pid int) ([]string, error) {
+	//$ cat /proc/17153/cgroup
+	//10:cpuset:/docker/2469451a734cd4699f15b090ce9d36554e89bcc913989907e8961d66a9aebc17
+	//9:perf_event:/docker/2469451a734cd4699f15b090ce9d36554e89bcc913989907e8961d66a9aebc17
+	//8:freezer:/docker/2469451a734cd4699f15b090ce9d36554e89bcc913989907e8961d66a9aebc17
+	//7:hugetlb:/docker/2469451a734cd4699f15b090ce9d36554e89bcc913989907e8961d66a9aebc17
+	//6:devices:/docker/2469451a734cd4699f15b090ce9d36554e89bcc913989907e8961d66a9aebc17
+	//5:memory:/docker/2469451a734cd4699f15b090ce9d36554e89bcc913989907e8961d66a9aebc17
+	//4:blkio:/docker/2469451a734cd4699f15b090ce9d36554e89bcc913989907e8961d66a9aebc17
+	//3:net_cls:/docker/2469451a734cd4699f15b090ce9d36554e89bcc913989907e8961d66a9aebc17
+	//2:cpuacct,cpu:/docker/2469451a734cd4699f15b090ce9d36554e89bcc913989907e8961d66a9aebc17
+	//1:name=systemd:/system.slice/docker.service
```

Comment:
I think it makes sense to move this parsing logic into https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/helpers_linux.go, if there isn't anything like it in that package.